marklogic-data-hub/src/main/resources/scaffolding/gradle_properties

```diff
@@ -1,127 +1,127 @@
-# mlAppServicesSimpleSsl=true
-
-
-mlStagingAppserverName=%%mlStagingAppserverName%%
-mlStagingPort=%%mlStagingPort%%
-mlStagingDbName=%%mlStagingDbName%%
-mlStagingForestsPerHost=%%mlStagingForestsPerHost%%
-mlStagingAuth=%%mlStagingAuth%%
-# To enable SSL for staging
-# mlStagingSimpleSsl=true
-
-mlFinalAppserverName=%%mlFinalAppserverName%%
-mlFinalPort=%%mlFinalPort%%
-mlFinalDbName=%%mlFinalDbName%%
-mlFinalForestsPerHost=%%mlFinalForestsPerHost%%
-mlFinalAuth=%%mlFinalAuth%%
-# To enable SSL for final
-# mlFinalSimpleSsl=true
-
-
-mlJobAppserverName=%%mlJobAppserverName%%
-mlJobPort=%%mlJobPort%%
-mlJobDbName=%%mlJobDbName%%
-mlJobForestsPerHost=%%mlJobForestsPerHost%%
-mlJobAuth=%%mlJobAuth%%
-# To enable SSL for jobs
-# mlJobSimpleSsl=true
-
-mlModulesDbName=%%mlModulesDbName%%
-mlModulesForestsPerHost=%%mlModulesForestsPerHost%%
-
-mlStagingTriggersDbName=%%mlStagingTriggersDbName%%
-mlStagingTriggersForestsPerHost=%%mlStagingTriggersForestsPerHost%%
-
-mlStagingSchemasDbName=%%mlStagingSchemasDbName%%
-mlStagingSchemasForestsPerHost=%%mlStagingSchemasForestsPerHost%%
-
-mlFinalTriggersDbName=%%mlFinalTriggersDbName%%
-mlFinalTriggersForestsPerHost=%%mlFinalTriggersForestsPerHost%%
-
-mlFinalSchemasDbName=%%mlFinalSchemasDbName%%
-mlFinalSchemasForestsPerHost=%%mlFinalSchemasForestsPerHost%%
-
-# You can override this to specify an alternate folder for your
-# custom forest info. Defaults to user-config/forests/
-# mlCustomForestPath=%%mlCustomForestPath%%
-
-# The name of the Role to create for Hub Access
-mlFlowOperatorRole=%%mlFlowOperatorRole%%
-mlFlowOperatorUserName=%%mlFlowOperatorUserName%%
-# this password is autogenerated for you via the 'gradle hubInit' task
-mlFlowOperatorPassword=%%mlFlowOperatorPassword%%
-
-# The name of the role to create for hub development
-mlFlowDeveloperRole=%%mlFlowDeveloperRole%%
-mlFlowDeveloperUserName=%%mlFlowDeveloperUserName%%
-mlFlowDeveloperPassword=%%mlFlowDeveloperPassword%%
-
-# The name of the role to create for hub deployment
-mlDataHubAdminRole=%%mlDataHubAdminRole%%
-
-# Deprecated property
-# If you are working with a load balancer please indicate so using
-# property "mlIsHostLoadBalancer"
-# When "mlIsHostLoadBalancer" is set to "true", the value specified for "mlHost" will be used as the load balancer.
-# You do not need to explicitly set the value of "mlLoadBalancerHosts" but if you do it must match the value of the property "mlHost"
-# mlLoadBalancerHosts=your-load-balancer-hostname
-
-# Default module permissions which allow flow-operator-role to execute flows
-mlModulePermissions=rest-reader,read,rest-writer,insert,rest-writer,update,rest-extension-user,execute,%%mlFlowDeveloperRole%%,read,%%mlFlowDeveloperRole%%,execute,%%mlFlowDeveloperRole%%,insert,%%mlFlowOperatorRole%%,read,%%mlFlowOperatorRole%%,execute
-
-# If DHF is running in a provisioned environment please specify it here
-# mlIsProvisionedEnvironment=false
+# These settings are used by Data Hub when

```

Comment:
Did anything change in this file? I can't tell because the entire file is marked as different.